Added content to Matches page

TeriDibbits opened the pull request:
Added default layout using Jekyll
Added card from the style tile html
stylized cards
added change button and content
resized images in images folder to fit cards

TeriDibbits committed:
Added content

Added default layout using Jekyll
Added card from the style tile html
stylized cards
added change button and content

MJholland reviewed (changes requested):
Everything looks good I just think the very last div needs a push-2 class instead so it's end space is more definitive.

MJholland reviewed (changes requested):
Also everything in the html should be wrapped in a main as part of best practices.

TeriDibbits committed:
changes requested

added main
change last div to push-2 as requested

TeriDibbits committed:
added media query

added media query to remove space on the side of the image in m-xl screens sizes

TeriDibbits commented:
I fixed the requested changes and added a media query to removed extra space on the image in larger screen sizes.
